spec/support/europeana_api_helper.rb

```diff
@@ -3,11 +3,28 @@ module EuropeanaAPIHelper
   RSpec.configure do |config|
     config.before(:each) do
       # API Search
+      
+      items = ''
+      
+      (0..20).each do |index|
+        id = '/sample/record' + index.to_s
+        items +=  '{"id":"' + id + '","title":["' + id + '"],"concepts":[],"proxies":[],"aggregations":[]}'
+        if(index < 20)
+          items += ','
+        end 
+      end
+      
+    
+    
+      log = Logger.new(STDOUT)
+      log.level = Logger::DEBUG
+      log.debug(items)
+      
       stub_request(:get, Europeana::API.url + '/search.json').
         with(query: hash_including(wskey: 'test')).
-        to_return(body: '{"success":true,"itemsCount":1,"totalResults":1,"items":[{"id":"/abc/123","title":["sample record"]}]}',
-                  status: 200,
-                  headers: { 'Content-Type' => 'text/json' })
+        to_return(body: '{"success":true,"itemsCount":' + items.size.to_s + ',"totalResults":' + items.size.to_s + ',"items":[' + items + ']}',
+        status: 200,
+        headers: { 'Content-Type' => 'text/json' })
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.